RELEASE_NOTES.md

```diff
@@ -1,6 +1,14 @@
 Release Notes
 =============
 
+v1.0.0-alpha.x
+--------------
+
+### Improvements
```

Comment:
[Fix] This needs to be a breaking change, as the resulting code has a new runtime requirement.

Reply:
https://github.com/OpenAssetIO/OpenAssetIO-TraitGen/pull/51/commits/722c4eca271ca23b6238b3a846f335d333a50b1e